### PR TITLE
Matt

### PR DESCRIPTION
**Pull request by mattvasil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,47 +9,48 @@ and curriculum, as well as update the calendar for their associates. Trainers
...
--Minh Thuy Nguyen\
--Alex Nowak\
--Douglas Ramirez\
--Donald Rowell\
--Matt Vasil\
--Alwyn Zhang\
--Christopher Zhang\
-
-###License 
+### Contributors:
```
Comment (DDRIV) [inline README.md:38]: I think you might have just looked at the participant list from today, because you are missing Jintao

**Review by cptvictor — DISMISSED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (cptvictor) [reply README.md:38]: Should you list all members here or should they be split up into front end and back end teams on their respective repos?

Comment (cptvictor) [reply README.md:38]: Side note, is there anybody taking care of the README on the front end side?

**Pull request closed without merging**